docs/guides/asr_getting_started_guide.md

```diff
@@ -0,0 +1,144 @@
+# ASR Getting Started Guide
+
+The ASR platform is a [Matter](https://github.com/project-chip/connectedhomeip)
+platform that uses ASR FreeRTOS SDK.
+
+---
+
+-   [ASR Getting Started Guide](#asr-getting-started-guide)
+    -   [Supported Chips](#supported-chips)
+    -   [Matter Example Applications](#matter-example-applications)
+    -   [Building the Example Application](#building-the-example-application)
+    -   [Commissioning](#commissioning)
+        -   [BLE mode](#ble-mode)
+        -   [IP mode](#ip-mode)
+    -   [Shell](#shell)
+    -   [OTA](#ota)
+
+---
+
+## Supported Chips
+
+The ASR platform is supported on:
+
+-   [ASR582X](http://www.asrmicro.com/en/goods/proinfo/36.html)
+-   [ASR595X](http://www.asrmicro.com/en/goods/proinfo/42.html)
+
+## Matter Example Applications
+
+Sample Matter applications are provided for the ASR platform. They can be used
+to speed up development:
+
+-   all-clusters-app
```

Comment:
consider not providing an exhaustive list as the list of apps may grow. Maybe something like an example and mentioning an sample apps with `/asr` subfolder are supported. Otherwise as we add new apps we have to maintain this list or the list becomes obsolete.